client/src/library/game-detail.tsx

```diff
@@ -25,34 +26,119 @@ export const LibraryGameDetail = ({
     navigate({ to: "/library" });
   };
 
-  const startNewGame = () => {
-    getLibraryService()
-      .createBlankStoryProgress({ storyKey: story.key })
-      .then((progress) =>
-        navigate({
-          to: "/game/$gameKey/$sceneKey",
-          params: { gameKey: story.key, sceneKey: story.firstSceneKey },
-          search: { storyProgressKey: progress.key },
-        }),
-      );
+  const createNewSave = async () => {
+    await getLibraryService().createBlankStoryProgress({ storyKey: story.key });
   };
 
+  const playCurrentGame = () => {
+    navigate({
+      to: "/game/$gameKey/$sceneKey",
+      params: {
+        gameKey: currentProgress.storyKey,
+        sceneKey: currentProgress.currentSceneKey,
+      },
+      search: { storyProgressKey: currentProgress.key },
+    });
+  };
+
+  // Progress percentage calculation using real number of scenes
+  const progressPercentage =
+    totalScenes > 0
+      ? Math.min(
+          Math.round((currentProgress.history.length / totalScenes) * 100),
+          100,
+        )
+      : 0;
+
   return (
-    <div className="flex h-full w-full flex-col items-center justify-start">
-      <GameBanner story={story} />
-      <div className="flex w-full flex-col gap-12 px-12 pt-12 md:flex-row">
-        <div className="md:w-6/12">
-          <GameInfo
-            story={story}
-            currentProgress={currentProgress}
-            otherProgresses={otherProgresses}
-            startNewGame={startNewGame}
-          />
-          <DeleteGameButton deleteGame={deleteGame} />
+    <div className="min-h-screen bg-gray-50">
+      <div className="mx-auto max-w-6xl px-6 py-8">
+        <div className="mb-12 flex gap-8">
+          <div className="flex-shrink-0">
+            {story.image ? (
+              <img
+                src={story.image}
+                alt={story.title}
+                style={{
+                  width: "calc(var(--spacing) * 104)",
+                  height: "calc(var(--spacing) * 78)",
+                }}
+                className="rounded-lg object-cover shadow-lg"
+              />
+            ) : (
+              <div
+                style={{
+                  width: "calc(var(--spacing) * 104)",
+                  height: "calc(var(--spacing) * 78)",
+                }}
+                className="flex items-center justify-center rounded-lg border-4 border-dashed border-gray-300 bg-gray-100"
+              >
+                <div className="flex h-20 w-20 items-center justify-center rounded-full bg-gray-300">
+                  <div className="h-12 w-12 rounded-full bg-gray-400"></div>
+                </div>
+              </div>
+            )}
+          </div>
+          <div className="flex-1 space-y-4">
+            <h1 className="text-4xl font-bold text-gray-800">{story.title}</h1>
+            {story.description && (
+              <p className="text-lg leading-relaxed text-gray-600">
```

Comment:
same qu'au-dessus sur les couleurs

```suggestion
              <p className="text-lg leading-relaxed text-muted-foreground">
```